distribution/lib/Standard/Table/0.0.0-dev/src/Table.enso

```diff
@@ -3991,3 +4019,10 @@ from_java_table java_table -> Table =
 ## PRIVATE
 Many_Files_List.from (that : Table) =
     Read_Many_Helpers.find_files_list_in_table that
+
+## PRIVATE
+private _column_ref_if_name table:Table column_name:Text prefix:Text=column_name ~on_fail=(Missing_Argument.throw column_name) =
```

Comment:
perhaps?
```suggestion
private _find_column_with_similar_name table:Table column_name:Text prefix:Text=column_name ~on_fail=(Missing_Argument.throw column_name) =
```

Reply:
Shall we move it to some other file, e.g. `Table_Helpers`? The `Table.enso` file is sooooo huge already I'd rather keep helper methods elsewhere if possible.